meson.build

```diff
@@ -38,6 +38,16 @@ if cpp.get_id() == 'msvc'
   add_project_arguments(msvc_args, language : 'cpp')
   # Disable SAFESEH with MSVC for libs that use external deps that are built with MinGW
   # noseh_link_args = ['/SAFESEH:NO']
+else
+  add_global_arguments('-fno-rtti', language : 'cpp')
+  add_global_arguments('-fno-exceptions', language : 'cpp')
+  add_global_arguments('-fno-threadsafe-statics', language : 'cpp')
+  if (true) # how to check it isn't mingw?
+    add_global_arguments('-fvisibility-inlines-hidden', language : 'cpp')
+  endif
+  if (false) # how to check it is ARM?
+    add_global_arguments('-mstructure-size-boundary=8', language : 'cpp')
```

Comment:
`if host_machine.cpu_family() == 'arm'` (for 32-bit arm, 64-bit is `aarch64` as per https://mesonbuild.com/Reference-tables.html#cpu-families

Reply:
Thank you :)